xbmc/windowing/gbm/DRMAtomic.cpp

```diff
@@ -119,6 +111,14 @@ void CDRMAtomic::FlipPage(struct gbm_bo *bo, bool rendered, bool videoLayer)
     }
   }
 
+  uint32_t flags = 0;
```

Comment:
why is this commit needed?

Reply:
I should probably add a better commit comment :)

I noticed a one time error with the `CDRMUtils::DrmFbGetFromBo(bo)` call when I was testing rockchip HDR stuff some weeks ago.
Looking at the code the `m_need_modeset` is reset on such error and next flip there is no modeset allowed.
With this change the modeset is just delayed until `DrmFbGetFromBo` is successful.

Reply:
makes sense. I just wanted to know the reason behind the change